CHANGELOG.md

```diff
@@ -1 +1,2 @@
 2015-11-07 - Initial release
+2015-12-17 - Adding authentication
```

Comment:
Can we include the version that introduced this change here as well as the date?